wordle-web/pages/game.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+    <v-card elevation="16" min-width="344">
+        <v-card-title>Hello Welcome to Pentagram</v-card-title>
+        <v-card-subtitle> This is our super basic wordle game called Pentagram. Penta for five, and gram for letter</v-card-subtitle>
+    </v-card>
+    <!--
+    <v-card class="mx-auto my-8" elevation="16" min-width="344">
```

Comment:
If you're gonna comment all this out, you could just delete it all lol

Reply:
True, not 100% sure why we commented it out in the first place, but deleting it would have accomplished the same thing.